src/rules/calc-functions-no-interpolation/README.md

```diff
@@ -0,0 +1,64 @@
+# calc-functions-no-interpolation
```

Comment:
@pamelalozano16 Sorry for not being clear about it, but this naming style won't work. 

If you look at the stylelint naming, it's done like this: `function-type-no-something`, e.g. `function-calc-no-unspaced-operator`.

<img width="474" alt="Screenshot 2023-09-13 at 23 34 02" src="https://github.com/stylelint-scss/stylelint-scss/assets/993108/2976cd30-245f-4882-90c3-d177120cba97">

Reply:
@pamelalozano16 the CSS spec seems to refer to the calc and other functions as "math-functions" (https://drafts.csswg.org/css-values/#math-function), but using that in the name might be confusing as Sass has it's own math functions

Reply:
Both the Sass docs and the CSS spec are referring to calculation functions, so maybe that's the name that would make most sense.

Reply:
So maybe the name could be something like `function-calculation-no-interpolation`.

Using `function-calc-no-interpolation` would be another option, but that would suggest that it would only work with the `calc()` function.

Reply:
Thanks for the clarification! `calculation` makes sense to me since both Sass docs and the CSS spec refer to these as calculation functions as you mentioned. :)